config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```
@@ -564,3 +564,28 @@ presubmits:
       testgrid-tab-name: capz-pr-ci-entrypoint-main
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
       description: Creates a CAPZ cluster and exports KUBECONFIG. This job validates ci-entrypoint.sh used by other repositories for running tests on CAPZ clusters.
+  - name: post-cluster-api-provider-azure-push-images-test
```

```suggestion
  - name: pull-cluster-api-provider-azure-push-images-test
```
nit (because of naming convention of presubmit jobs)